### ISSUE TITLE
<Modal/> - tab key not going through the inputs (accessibility) 

### ISSUE THREAD

**Issue by alexandraseri** [wix/wix-style-react#5436]
# 🐛 Bug Report

### 🏗 Relevant Components
Modal

### 😯 Current Behavior

When using Tab key to go over the modal (accessibility) - it does not work

### 🤔 Expected Behavior

Being able to go from input to input to buttons using Tab in Modal.

### 👣 Steps to Reproduce

```
class MyComponent extends React.Component {
  constructor() {
    this.state = {
      open: false,
    };
  }
  closeModal() {
    this.setState({open: !this.state.open});
  }
  render() {
    return (
    <>
    <Button onClick={this.closeModal.bind(this)}>Open Modal</Button>
    <Modal isOpen={this.state.open} onRequestClose={() => this.setState({open: false})}>
    <MessageBoxFunctionalLayout>
        <Row>
          <Col span="3" />
          <Col span="6">
            <Card>
              <Card.Content>
                <Container fluid>
                  <Row>
                    <Col>
                      <FormField label="Name">
                        <Input />
                      </FormField>
                    </Col>
                  </Row>
                  <Row>
                    <Col>
                      <FormField label="Title">
                        <Dropdown
                          options={[
                            { id: 0, value: 'Mr' },
                            { id: 1, value: 'Ms' },
                          ]}
                        />
                      </FormField>
                    </Col>
                  </Row>
                  <Row>
                    <Col>
                      <Box align="right">
                        <Box marginRight={2}>
                          <Button priority="secondary" onClick={this.closeModal.bind(this)}>Cancel</Button>
                        </Box>
                        <Button onClick={this.closeModal.bind(this)}>Submit</Button>
                      </Box>
                    </Col>
                  </Row>
                </Container>
              </Card.Content>
            </Card>
          </Col>
        </Row>
      </MessageBoxFunctionalLayout>
      </Modal>
      </>
    );
  }
}
```

  - WSR version: 8.6.0
  - Yoshi version: 4.65.0

### 👀 Severity

Major - accessibility is impacted

**Comment by EgoziE** [comment]
Fixed in 8.17.0
